### ISSUE TITLE
Adjust list command with pretty print format

### ISSUE THREAD

**Issue by Bajger** [axlgithub/CommandLineLauncher#28]
_Currently, list commands lists only sequence nr. and name of image and VM (or template). This is insufficient, when compared to UI version._

**Describe the request**
Lists commands should output pretty print format with useful information (as described [here](https://github.com/PharoLauncherCmdLine/CommandLineLauncher/commit/a6c725cc3a3dadd02e8572a8ab05b14a992d3ee2#diff-c716a6a7c64224417d663d79fa12fc5caddbfe925600c575e274ad66778a9fc6)

**Expected behavior**
Image and VM list commands should output following:
- Image: nr., Image name, Architecture, Pharo Version, Last Modified, Last Modified ago
- VM:  nr., VM name, Updated, Remote URL

2) provide additional flags for print format
- short version of output using `brief` flag that will print just name
- `delimiter` flag for custom delimiter when printing image or VM attributes
- `rowMode` - prints one attribute per line only

**Expected development cost**
adjust `list:` method for listing attributes in table format and in block mode
